src/promptflow-azure/promptflow/azure/operations/_run_history_client.py

```diff
@@ -0,0 +1,92 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
```

Comment:
shall we put all these client into a separte folder? as they are not operations

Reply:
I will group them to a util folder in the next pr.